packages/cardano-services/test/Asset/CardanoTokenRegistry.test.ts

```diff
@@ -19,7 +19,7 @@ describe('CardanoTokenRegistry', () => {
     it('complete attributes', () =>
       expect(
         toCoreTokenMetadata({
-          decimals: { value: 23 },
+          decimals: { value: 0 },
```

Comment:
The bug case was with a falsy decimals value so the test is updated accordingly to catch it